qt/python/mantidqtinterfaces/mantidqtinterfaces/dns_powder_tof/main_presenter.py

```diff
@@ -71,6 +72,13 @@ def _tab_changed(self, old_tab_index, tab_index):
         for observer in self._parameter_abo.observers:
             if observer.view == actual_view:
                 self._parameter_abo.notify_focused_tab(observer)
+        if self.modus.name == "single_crystal_elastic":
+            if tab_index == 4:
+                self.modus.widgets["plot_elastic_single_crystal"].view.views_menu.menuAction().setVisible(True)
```

Comment:
Just a suggestion, but it might be preferable to hide some of this (on subsequent lines as well) in a view method called `set_view_menu_visibility` or similar - that way it would be easier to mock in testing.
```suggestion
                self.modus.widgets["plot_elastic_single_crystal"].view.set_view_menu_visibility(True)
```

Reply:
Similar methods are used here as well
https://github.com/mantidproject/mantid/blob/4561be9e3143dea1cb643090c1b28a3e337ab6d8/qt/python/mantidqtinterfaces/mantidqtinterfaces/dns_single_crystal_elastic/plot/elastic_single_crystal_plot_view.py#L71-L72